pkgs/games/hexgui/default.nix

```
@@ -0,0 +1,34 @@
+{ ant
+, fetchFromGitHub
+, jdk
+, lib
+, makeWrapper
+, stdenv
+}:
+stdenv.mkDerivation {
+  pname = "hexgui";
+  version = "unstable-2022-5-30";
+
+  src = fetchFromGitHub {
+    owner = "selinger";
+    repo = "hexgui";
+    rev = "d94ce1d35a22dad28d3e7def4d28e6bebd54da9d";
+    hash = "sha256-1MroFH2JSEZHFigcsw1+xyHJWEnHTvHmRPVirUgwM6I=";
+  };
+
+  nativeBuildInputs = [ ant jdk makeWrapper ];
+  buildPhase = "ant";
+
+  installPhase = ''
+    mkdir $out;
+    cp -r bin lib $out
+    wrapProgram $out/bin/hexgui --prefix PATH : ${jdk}/bin
```

Why do we need to wrap it with jdk? Does it exec to that?

```suggestion
    wrapProgram $out/bin/hexgui --prefix PATH : ${lib.makeBinPath [ jdk ]}
```